src/app/server/JointFabricDatastore.cpp

```diff
@@ -531,12 +586,12 @@ JointFabricDatastore::AddBindingToEndpointForNode(
     newBindingEntry.nodeID            = nodeId;
     newBindingEntry.endpointID        = endpointId;
     newBindingEntry.binding           = binding;
-    newBindingEntry.statusEntry.state = Clusters::JointFabricDatastore::DatastoreStateEnum::kCommitted;
+    newBindingEntry.statusEntry.state = Clusters::JointFabricDatastore::DatastoreStateEnum::kPending;
```

Comment:
```suggestion
    newBindingEntry.listID = highestListID + 1;
    newBindingEntry.statusEntry.state = Clusters::JointFabricDatastore::DatastoreStateEnum::kPending;
```

Reply:
Updated to incorporate the suggestion while retaining the existing format for generating ListIDs